### ISSUE TITLE
Train on a dataset and test on another

### ISSUE THREAD

**Issue by SofianeB-03** [edwardzhou130/PolarSeg#40]
Hello, thank you very much for your great work ! I have a question: when we test the pre-trained model on a new database, should we modify the axes of the point cloud such that they are the same as the training database ? For example if I test on Nuscenes the pre-trained PolarSeg on Semantickitti, should I modify the axes of Nuscenes according those of SemanticKitti ?

**Comment by edwardzhou130** [comment]
It's not necessary to modify the axis of the point cloud if the only difference between your dataset and KITTI is the rotation of the axis. The bigger problem is the number of class and class definitions. You need to carefully match the class in your dataset to the KITTI dataset to get a meaningful prediction. And some other differences like the number of points/ range also could have a big influence on the result.

**Comment by SofianeB-03** [comment]
Thanks for your reply ! A last question: is it because the data has been augmented (translation + rotation) during training that we do not need to modify the axes of the point cloud?

**Comment by edwardzhou130** [comment]
Yes, we did the rotation and flip augmentation in training.

**Comment by SofianeB-03** [comment]
Thank you !
